build.sbt

```diff
@@ -8,7 +8,7 @@ val compileDependencies = Seq(
   filters,
   "com.typesafe.play"  %% "play"               % PlayVersion.current,
   "com.google.guava"   %  "guava"              % "19.0",
-  "uk.gov.hmrc"        %% "http-verbs"         % "8.10.0-play-25",
+  "uk.gov.hmrc"        %% "http-verbs"         % "9.0.0-play-25",
```

Comment:
Could we do the latest or 9.1.0 instead?

Reply:
@konradwudkowski - Yes that works too. I have updated the pull request.